taxi/views.py

```diff
@@ -1,8 +1,9 @@
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import render
-from django.urls import reverse_lazy
+from django.shortcuts import render, redirect, get_object_or_404
+from django.urls import reverse_lazy, reverse
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
+from taxi.forms import DriverLicenseUpdateForm, DriverCreationForm
 
```

Comment:
split different groups of imports

Reply:
This is not done - you have to group imports from django and imports from your app together and separate them with blank line